### ISSUE TITLE
App locks up on sandbox file

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry-desktop#2508]
## The Issue
When opening lbry://HartwigChessSet#690cf3aa77e89b9eb58ec5e761bfa9455093d395 (make sure it's not downloaded already), the app will lock up. You'll get a Quit/Wait dialog after 30 seconds or so.  If you quit/restart, and then reopen, it works. 

## System Configuration

- LBRY Daemon version:
- LBRY App version:
- LBRY Installation ID:
- Operating system:

**Comment by btzr-io** [comment]
Looks like is stuck on `waiting for blob...`, also got this error on the console:
```
Not allowed to load local resource: blob:file:///dda0d7dd-9962-4e40-901f-e4391fa14fe5
```
> Note: That's not a 3D file, so it's not an issue with the 3D-viewer

**Comment by btzr-io** [comment]
Render media don't have support for the `lbry-x` format, so I'm not sure why is it here ?
https://github.com/lbryio/lbry-desktop/blob/d9e9a95879650b1065f1a27d844e3d1b887517dd/src/ui/component/fileViewer/internal/player.jsx#L260

**Comment by btzr-io** [comment]
@tzarebczan I managed to suppress the error / blocking of the app, now the issue is displaying the content once the download is completed :+1:

**Comment by tzarebczan** [comment]
Hmm, it should do that by default for these file types. I can take a look in your branch to see why it's not. I know others of a similar type would work...

**Comment by btzr-io** [comment]
@tzarebczan No, there was no real logic for that :upside_down_face:,
Must've been removed somewhere along the way ?

**Comment by btzr-io** [comment]
Fixed on #2530
